### PR TITLE
Multiple master playlists

### PR DESCRIPTION
**Pull request by CarlinWilliamson**
## Context

[Corresponding Pivotal Tracker story](https://www.pivotaltracker.com/story/show/175614136)

## Description 💬
Playlist selection now has checkboxes. The user can pick any number of them.
The selected playlists are combined into one massive playlist which is used as the master

## Checklist ✅:

- [x] I assigned the story to myself on Pivotal Tracker before I started working on this
- [x] Added a link to the user story context
- [x] Added a description of my changes
- [x] Assigned at least two reviewers

### CODE REVIEW

File: src/components/MasterPlaylist.tsx [hunk 1]
```diff
@@ -63,17 +63,17 @@ export default function MasterPlaylist(props: { playlist: PlaylistObj, usedTrack
       .map((t: TrackObj) => t.genres)
       .flat()
       .forEach((g: string) => map.set(g, (map.get(g) ?? 0) + 1));
-      
+
     var mapAsc = Array.from(map.entries()).sort((a, b) => b[1] - a[1]);
     mapAsc.splice(3, 99999);
```
Comment (z5160345) [inline src/components/MasterPlaylist.tsx]: dont know whose code this is but the hard coded 99999 screams "break me" lmao. Doubt itll be a problem tho

**Review by z5160345 — COMMENTED** [review]
Would be nice to be able to reselect playlists instead of having to log out then log back in

File: src/components/MasterPlaylist.tsx (hunk 1 above)
Comment (CarlinWilliamson) [reply src/components/MasterPlaylist.tsx]: It's mine from a while ago. I wasn't bothered to look up the ts equivalient of `Integer.MAX_VALUE`.

It's been fixed

**Comment by CarlinWilliamson** [comment]
> Would be nice to be able to reselect playlists instead of having to log out then log back in

This has been implemented

**Review by featherbear — APPROVED** [review]
lgtm

**Review by z5160345 — APPROVED** [review]
LGTM

**Pull request merged**